### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by jaygambetta**
Finishing getting started notebook. This is part of issue #293

### CODE REVIEW

**Comment by jaygambetta** [comment]
What circuit drawing.  If it is the latex I can’t get that working at the moment.

**Comment by jaygambetta** [comment]
@nonhermitianI think I fixed your comments. About the circuits I'm not sure what you mean

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 
-Qconfig.py
```
[the remark this answers is not in the document]
Comment (jaygambetta) [reply .gitignore]: sure add it back dont show @nonhermitian  :-)

File: qiskit/basics/getting_started.ipynb [hunk 2]
```diff
@@ -48,47 +48,28 @@
...
-    "\n",
-    "### Building your circuit: Create it  <a id='sectionB'></a>"
+    "The basic concept of Qiskit is an array of quantum circuits. The workflow consists of two stages: [Build](#sectionB), and [Execute](#sectionR). Build allows you to make different quantum circuits that represent the problem you are solving; Execute allows you to run them to run on different backends.  After the jobs have been run, the data is collected. There are methods for putting this data together, depending on the program. This either gives you the answer you wanted, or allows you to make a better program for the next instance."
```
Comment (nonhermitian) [inline qiskit/basics/getting_started.ipynb]: "Execute allows you to run them to run on different backends" should go to "Execute allows you to run them on different backends "

**Comment by ajavadia** [comment]
Ok made some small fixes (including the one @nonhermitian said). Good to go.

**Pull request merged**